### PR TITLE
2nd Pull Request for "Displaying and Inputting data project"

### PR DESCRIPTION
**Pull request by jose-Abel**
This is the 2nd Pull Request for the Microverse HTML and CSS 2nd project, the first one was rejected because the Readme was pushed to the master branch, and that is a wrong GitHub flow, now it wasn't actually pushed to the master branch, it was created on the master branch and then created on the signup branch as well, we thought that it was fine to do it like that just for the Readme file. However, after being pointed that it was wrong GitHub flow procedure, I was told that I could delete that Readme file from the master and resubmit for a code review and I just did that on this Pull Request.

Hoping to get another amazing feedback!!

### CODE REVIEW

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @jose-Abel  and @Ceci007! 👋👋 
You have done a great job with your project so far!👍 It looks amazing and your code looks really clean!!👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] There are many instances where some of your code selectors do the same things. I have highlighted some areas where those selectors can be grouped together. Please use my comments as a guide but search through your code for any additional instances of this. 

- [x] I noticed that you have not added the code linters to your project. You can find instructions on how to add them [here](https://github.com/microverseinc/linters-config/tree/master/html-css). 
Screen Shot 2020-07-10 at 8 21 53 PM


****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+  width: 2.5rem;
+  height: 2.5rem;
+  margin-right: 0.2rem;
+}
+
+.mint-logo img, .quickbooks-logo img, .turbotax-logo img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
```
Comment (BrittanyBlake) [inline style.css]: The code on these lines can be grouped with the codes on the following lines as they do the same thing:

- lines 97-101

File: style.css [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+.logo-box {
+  width: 13rem;
+  height: auto;
+}
+
+
+
+.logo-box a {
+  text-decoration: none;
+}
```
Comment (BrittanyBlake) [inline style.css]: The code on these lines can be grouped with the codes on the following lines as they do the same thing:

- lines 117-120

File: style.css [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-top {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (BrittanyBlake) [inline style.css]: The code on these lines can be grouped with the codes on the following lines as they do the same thing.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+
+.header-text p {
+  color: grey;
+}
+
+.header-text a {
+  text-decoration: none;
+  color: var(--color-primary);
+}
```
Comment (BrittanyBlake) [inline style.css]: The code on these lines can be grouped with the codes on the following lines as they do the same thing:

- lines 169-172

- lines  252-254

- lines 272-275

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 
#### The project looks really good! Congratulations

Just one detail, you could add more sections to your readme file. Check [this](https://github.com/microverseinc/readme-template) template (especially for the next projects). Also fix the conflicts on this branch so you will be able to merge this branch.

#### You did really good on this project. You can merge now.

Good Luck and Happy Coding :+1: 

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**